### DISCUSSION TITLE
v3.4.0

### DISCUSSION THREAD

**Discussion by sojan-official** [chatwoot/chatwoot#8584] [Releases & Announcements]
## Changelog

- Ability to manage premium self-hosted plans from super admin pannel
- Ability to manage white label settings from super admin pannel ( enterprise )
- Support for stickers and outgoing attachments in line channel
- Support for more sort options in conversation dashboard
- Support for custom attributes in message variables
- Delivery reports for Facebook channel
- Support for Authentication in case of Twilio media messages
- Multiple performance improvements and bug fixes

Thanks to @clairton, @soymgomez, @Toinne, @hugodeco, @LiamAshdown, @mazenkhalil,   for the contributions

<hr /><em>This discussion was created from the release <a href='https://github.com/chatwoot/chatwoot/releases/tag/v3.4.0'>v3.4.0</a>.</em>

**Comment by PuseTV-Repository** [comment]
How can i upgrade?

**Reply by sojan-official** [reply]
https://www.chatwoot.com/docs/self-hosted/deployment/upgrade

**Comment by angelgnz** [comment]
Hi there,

After the recent update, incoming media messages in the WhatsApp inbox are not loading.

Details:

Using: Docker image
Inbox: WhatsApp (Twilio)
Error: Down::ClientError ([400 Bad Request](https://www.twilio.com/docs/api/errors/400))
I'm not certain if this [PR](https://github.com/chatwoot/chatwoot/pull/8396) is causing the issue, as rolling back to the previous version resolves the issue.

Thanks!

**Reply by sojan-official** [reply]
@angelgnz  could you enable HTTP authentication for Twilio media

**Reply by angelgnz** [reply]
Now i feel dumb 😄 

Thanks!!!

[Enable HTTP Auth ](https://support.twilio.com/hc/en-us/articles/223183748-Protect-Media-Access-with-HTTP-Basic-Authentication-for-Programmable-Messaging)

**Comment by danilocampos** [comment]
Hello! I always manage to update ChatWoot on Heroku through my mirror repository, but it's not working with this version. Could you guide me through it?
<img width="837" alt="Screenshot at Dec 21 06-29-25" src="https://github.com/chatwoot/chatwoot/assets/15961144/0ab5de6d-193d-4105-b840-44eb75bbf564">

**Comment by Double-Click-ERP** [comment]
> ## Changelog
> * Ability to manage premium self-hosted plans from super admin pannel

what is the plans / prices for self-hosted ?

**Comment by maccup** [comment]
Regarding the new plans, it seems like I'm on the community edition plan (info from super admin).

But when I go to Stripe, I can see:
Self Hosted - Premium $0.00 per month

How does it work? To have more features like branding, etc. Is it necessary to have a paid plan? Or should I somehow configure Stripe on my end?

<img width="700" alt="Screenshot 2023-12-26 at 10 55 09" src="https://github.com/chatwoot/chatwoot/assets/57940197/6b75e2d3-7861-46e0-8db4-db6ae145842a">
<img width="1475" alt="Screenshot 2023-12-26 at 10 55 00" src="https://github.com/chatwoot/chatwoot/assets/57940197/b8af4509-4ebc-4615-884d-b13d93053734">

**Reply by Rannielson** [reply]
Can you clear this doubt for us? @sojan-official
